src/main/java/com/example/KitchenProject/RecipeController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.KitchenProject;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RestController
+public class RecipeController {
+    List <Recipe> recipes = Arrays.asList(
+            new Recipe("parówki", Arrays.asList(new Ingredient("parówka",4,QuantityUnit.GRAM)), "domowy", 10, CookingLevel.MASTER),
```

Comment:
Those 3 lines are way too long :D Here are few useful settings for you to turn on in Intellij:
Settings -> Code Style -> Java -> Wrapping and Braces -> 
1. Ensure right margin is not exceeded, 
2. Method call arguments: New line after '(', Place ')' on new line

and here is how I would format this piece of code:
```java
    List<Recipe> recipes = Arrays.asList(
            new Recipe(
                    "parówki",
                    Arrays.asList(new Ingredient("parówka", 4, QuantityUnit.GRAM)),
                    "domowy",
                    10,
                    CookingLevel.MASTER
            ),
            new Recipe(
                    "tosty",
                    Arrays.asList(
                            new Ingredient("chleb tostowy", 2, QuantityUnit.CUP),
                            new Ingredient("szynka", 1, QuantityUnit.LITER),
                            new Ingredient("ser", 1, QuantityUnit.SPOON)
                    ),
                    "domowy",
                    5,
                    CookingLevel.EASY
            ),
            new Recipe(
                    "chrupki z mlekiem",
                    Arrays.asList(
                            new Ingredient("chrupki", 100, QuantityUnit.GRAM),
                            new Ingredient("mleko", 300, QuantityUnit.MILLILITER)
                    ),
                    "starożytny",
                    15,
                    CookingLevel.MEDIUM
            )
    );
```
Don't you think it's way more readable that way?:) 
For me, the rule of thumb is that if the line is too long, each function argument should be on the new line.